plaso/parsers/safari_cookies.yaml

```diff
@@ -0,0 +1,121 @@
+name: binarycookies
+type: format
+description: Safari cookies file format
+urls: ["https://github.com/libyal/dtformats/blob/master/documentation/Safari%20Cookies.asciidoc"]
+---
+name: byte
+type: integer
+attributes:
+  format: unsigned
+  size: 1
+  units: bytes
+---
+name: uint16
+type: integer
+attributes:
+  format: unsigned
+  size: 2
+  units: bytes
+---
+name: uint32
+type: integer
+attributes:
+  format: unsigned
+  size: 4
+  units: bytes
+---
+name: uint32be
+type: integer
+attributes:
+  byte_order: big-endian
+  format: unsigned
+  size: 4
+  units: bytes
+---
+name: uint64
+type: integer
+attributes:
+  format: unsigned
+  size: 8
+  units: bytes
+---
+name: float64
+type: floating-point
+attributes:
+  size: 8
+  units: bytes
+---
+name: cstring
+type: string
+encoding: ascii
+element_data_type: byte
+elements_terminator: "\x00"
+---
+name: binarycookies_file_header
+type: structure
+attributes:
+  byte_order: big-endian
+members:
+- name: signature
+  type: stream
+  element_data_type: byte
+  number_of_elements: 4
+- name: number_of_pages
+  data_type: uint32
+---
+name: binarycookies_page_sizes
+type: sequence
+element_data_type: uint32be
+number_of_elements: binarycookies_file_header.number_of_pages
+---
+name: binarycookies_file_footer
```

Comment:
Please remove the footer, as it's not used

Reply:
ack